### PR TITLE
Refactor of animation zone

### PR DESCRIPTION
**Pull request by Adityamaurya123uma**
Refactor of animation zone

### CODE REVIEW

**Comment by Adityamaurya123uma** [comment]
@Deependra3010 Can you please have a look at this PR. I had refactored the animation zone with better animation.

**Comment by Adityamaurya123uma** [comment]
@Deependra3010 Can you recheck now. I just added another commit.

**Comment by Deependra3010** [comment]
> @Deependra3010 Can you recheck now. I just added another commit.

Better Now could you have a look at speed, at minimum speed 1 also its too fast it does not event take 1sec to sort.

**Comment by Adityamaurya123uma** [comment]
@Deependra3010 Yes, the speed, actually it work reverse at the moment, meaning 1  indicates fast and 10 indicates slow.
I'll handle it.

**Comment by Deependra3010** [comment]
@Adityamaurya123uma Just a few issue's and some suggestion otherwise look's good!!!

**Review by Deependra3010 — CHANGES_REQUESTED** [review]

File: algorithm-visualizer/src/components/content/animationzone.js [hunk 1]
```diff
@@ -1,81 +1,116 @@
...
+        />
+        <div className="speed-label">Speed: {speed}</div>
+        <input
+          type="range"
+          min="1"
+          max="25"
+          value={this.state.arraySize}
+          onChange={(e) => this.setState({ arraySize: e.target.value })}
+          onInput={() => this.handleArraySizeChange()}
+        />
```
[the remark this answers is not in the document]
Comment (Adityamaurya123uma) [reply algorithm-visualizer/src/components/content/animationzone.js:109]: I can do this in future commits once the animation zone is fully built. I added todo for now, that would remind me later

**Comment by Adityamaurya123uma** [comment]
@Deependra3010 All the changes suggested by you are done.

**Review by Deependra3010 — APPROVED** [review]

**Pull request merged**